src/services/mangoTransactions/index.js

```diff
@@ -0,0 +1,17 @@
+export const sumMangos = (mangoTransactions) => {
+  if (mangoTransactions) {
+    return mangoTransactions.reduce((acc, curr) => {
+      return acc + curr.mangoCount;
+    }, 0);
+  }
+  return 0;
+};
+
+export const addMangoTransaction = (
+  mangoTransactions,
+  user_id,
+  mangoAmount
+) => {
+  // TODO implement
```

Comment:
Looks like it's done

Reply:
oh lmao u right haha
this was actually a bit of a throw-away-but-i-know-where-we-can-use-this function to fix that dumb eslint error about default exporting if there's only one thing be exported in file sigh